test/app.test.js

```diff
@@ -740,7 +740,7 @@ describe('app', function() {
 
       expect(function() {
         app.dataSource('bad-ds', { connector: 'throwing' });
-      }).to.throw(/bad-ds.*throwing/);
+      }).to.throw(/bad-ds.*expected test error/);
```

Comment:
This is suspicious, as it does not relate to the rest of the changes made in this patch. Could you please check? If the tests are failing on the current master, then let's fix them in a different pull request.

In this particular test, it's important to check that the data source name (`throwing`) is included in the error message. The connector error (`expected test error`) is already included by loopback-datasource-juggler.